### DISCUSSION TITLE
Non ASCII output is garbled

### DISCUSSION THREAD

**Discussion by dlemstra** [ImageMagick/ImageMagick#6695] [Contribute]
### Discussed in https://github.com/ImageMagick/ImageMagick/discussions/3962

<div type='discussions-op-text'>

<sup>Originally posted by **sergeevabc** July 19, 2021</sup>
Windows 7, ImageMagick 7, [орлы.png](https://user-images.githubusercontent.com/373718/126172617-257f61cf-500b-48b2-ae32-54c4a78459a6.png)

```
$ magick.exe -verbose орлы.png орлы.webp
РѕСЂР»С‹.png PNG 162x192 162x192+0+0 8-bit Gray 11768B 0.000u 0:00.024              
РѕСЂР»С‹.png=>РѕСЂР»С‹.webp PNG 162x192 162x192+0+0 8-bit sRGB 6428B 0.031u 0:00.091
^^^^^^^^
```
</div>

Printing of the filename seems to be incorrect on Windows. The problem is most likely that `_vfwprintf_l` should be used inside `FormatLocaleFileList` but I was unable to find a proper fix for this.

**Comment by losito105** [comment]
Hello! I'd like to try to resolve this issue.

**Comment by dlemstra** [comment]
Sure go ahead and please let us know if you need any help.

**Comment by losito105** [comment]
Will do, thank you!
